packages/office-ui-fabric-react/src/components/Panel/examples/Panel.HiddenOnDismiss.Example.tsx

```diff
@@ -29,7 +26,7 @@ export class PanelHiddenOnDismissExample extends React.Component<
     this.setState({ showPanel: true });
   };
 
-  private _hidePanel = (): void => {
+  private _closePanel = (): void => {
```

Comment:
`_hidePanel` is actually a better name in this case, since the panel is hidden not actually closed

Reply:
Based on your previous comment to I think I agree, I'll update all `_closePanel` to `_hidePanel` as it does better reflect what is happening.

Reply:
That's not quite what I meant--this case does actually do something different than the others, so I think a different name would be appropriate. Usually, the panel does in fact close (disappears and all its DOM is destroyed), but in this case it's *only* hidden. (The flag was added for cases that need to maintain internal panel state between times it's displayed.)

Reply:
Good point, changing those functions to be `_dismiss(Panel|Dialog)`.